packages/jupyter-ai/jupyter_ai/chat_handlers/base.py

```diff
@@ -58,6 +58,15 @@ class BaseChatHandler:
 
     routing_type: HandlerRoutingType = ...
 
+    uses_llm = ...
+    """Class attribute specifying whether this chat handler uses the LLM
+    specified by the config. Subclasses must define this. Should be set to
+    `False` for handlers like `/help`."""
```

Comment:
A few (hopefully helpful) questions:
- should it be type-annotated as `ClassVar[bool]`?
- would it possibly make sense to default it to `True` to avoid "breaking" existing custom providers?
- if not, is it worth adding it to the example in the docs?

https://github.com/jupyterlab/jupyter-ai/blob/54daef863f6a341c2fd886af3609c1f80481c834/docs/source/developers/index.md?plain=1#L140-L144

Reply:
Thanks for the suggestion. I've added a type annotation and a default value of `True`. 

The reason we use the ellipses is because this is the recommended way in Pydantic v1 to declare a "required class attribute" that subclasses should override. However, Pydantic v1 doesn't even enforce this convention, so I'm OK with setting a default here.